influxdb_migration_1_to_timestream_for_influxdb_guide.md

```diff
@@ -0,0 +1,281 @@
+
+#### Create a 1.x-compatible authorization
+
+Use the Influx CLI `influx v1 auth create` command to create a 1.x-compatible authorization that grants read/write permissions to specific Timestream for InfluxDB buckets. Provide the following:
+
+- List of bucket IDs to grant read or write permissions to.
+- New v1 auth username.
+- New v1 auth password (when prompted).
+
+```shell
+# Single bucket
+
+influx v1 auth create \
+  --host <Timestream for InfluxDB endpoint> \
+  --token <token> \
+  --read-bucket 00xX00o0X001 \
+  --write-bucket 00xX00o0X001 \
+  --username example-user
+
+# Multiple buckets
+
+influx v1 auth create \
+  --host <Timestream for InfluxDB endpoint> \
+  --token <token> \
+  --read-bucket 00xX00o0X001 \
+  --read-bucket 00xX00o0X002 \
+  --write-bucket 00xX00o0X001 \
+  --write-bucket 00xX00o0X002 \
+  --username example-user
+```
+
+For information about managing 1.x compatible authorizations, see the [`influx v1 auth` command documentation](https://docs.influxdata.com/influxdb/v2/reference/cli/influx/v1/auth/).
+
+## Migrate time series data
+
+To migrate time series data from your InfluxDB 1.x instance to Timestream for InfluxDB:
+
+1. On the InfluxDB 1.x instance, use the InfluxDB 1.x [`influx_inspect export` command](https://docs.influxdata.com/influxdb/v1/tools/influx_inspect/#export) to export time series data as line protocol. Include the `-lponly` flag to exclude comments and the data definition language (DDL) from the output file.
+
+   *We recommend exporting each DBRP combination separately to easily write data to a corresponding InfluxDB 2.7 bucket*.
+
+   ```shell
+    # Syntax
+    influx_inspect export \
+      -database <database-name> \
+      -retention <retention-policy-name> \
+      -out <output-file-path> \
+      -lponly
+
+    # Example
+    influx_inspect export \
+      -database example-db \
+      -retention example-rp \
+      -out /path/to/example-db_example-rp.lp \
+      -lponly
+   ```
+
+2. Use the Influx CLI `influx write` command to write the exported line protocol to Timestream for InfluxDB.
+   ```shell
+    # Syntax
+    influx write \
+      --host <Timestream for InfluxDB endpoint> \ # For example, https://my-endpoint:8086
+      --org <org> \
+      --token <token>
+      --bucket <bucket-name> \
+      --file <path-to-line-protocol-file>
+
+    # Example
+    influx write \
+      --host <Timestream for InfluxDB endpoint> \
+      --org <org> \
+      --token <token>
+      --bucket example-db/example-rp \
+      --file /path/to/example-db_example-rp.lp
+   ```
+
+3. Repeat steps 1-2 for each bucket.
+
+## Migrate continuous queries
+
+For information about migrating InfluxDB 1.x continuous queries to Timestream for InfluxDB tasks, see InfluxData's [Migrate continuous queries to tasks](https://docs.influxdata.com/influxdb/v2/install/upgrade/v1-to-v2/migrate-cqs/) guide.
```

Comment:
The guide for migrating continuous queries is massive. I thought it would be better to follow what the InfluxDB 1.x to 2.7 guide does and simply link to the guide for migrating continuous queries.